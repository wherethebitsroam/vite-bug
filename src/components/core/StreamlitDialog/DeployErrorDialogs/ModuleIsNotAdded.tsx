import React from "react"
import { IDeployErrorDialog } from "./types"

function ModuleIsNotAdded(module: string): IDeployErrorDialog {
  return {
    title: "Unable to deploy app",
    body: (
      <>
        <p>
          The file <code>{module}</code> has not been added to the repo.
        </p>
        <p>Please add it and push to GitHub to continue.</p>
      </>
    ),
  }
}

export default ModuleIsNotAdded
