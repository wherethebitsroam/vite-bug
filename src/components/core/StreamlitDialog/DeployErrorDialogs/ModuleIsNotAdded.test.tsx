import ModuleIsNotAdded from "./ModuleIsNotAdded"

describe("DeployErrorDialog - ModuleIsNotAdded", () => {
  it("should render without crashing", () => {
    const dialog = ModuleIsNotAdded("module")

    expect(dialog).toMatchSnapshot()
  })
})
