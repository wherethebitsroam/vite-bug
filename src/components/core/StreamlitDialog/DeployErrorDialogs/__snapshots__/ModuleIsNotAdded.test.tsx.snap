// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeployErrorDialog - ModuleIsNotAdded should render without crashing 1`] = `
Object {
  "body": <React.Fragment>
    <p>
      The file 
      <code>
        module
      </code>
       has not been added to the repo.
    </p>
    <p>
      Please add it and push to GitHub to continue.
    </p>
  </React.Fragment>,
  "title": "Unable to deploy app",
}
`;
